src/common.rs

```diff
@@ -29,6 +29,7 @@ pub enum Architecture {
     Wasm32,
     Wasm64,
     Xtensa,
+    CSKY,
```

Comment:
```suggestion
    Csky,
```
Rustc warns about non-CamelCase names.